h2o-core/src/main/java/hex/grid/hyperspace/HyperSpaceSearchCriteria.java

```diff
@@ -1,4 +1,4 @@
-package hex.grid;
```

Comment:
any backwards compatibility concern?
I think this is part of Java `API`

Reply:
Oh, is it ? Then we should leave it as-is I guess and not refactor it ever, or more precisely ion a backwards compatible way. If yes, I'll revert it.